src/database/model/computational_asset/accelerator.py

```diff
@@ -0,0 +1,48 @@
+from typing import TYPE_CHECKING
+from database.model.field_length import NORMAL
+from sqlmodel import Field, SQLModel, Relationship
+
+if TYPE_CHECKING:  # avoid circular imports; only import while type checking
+    from database.model.computational_asset.computational_asset import ComputationalAsset
+
+
+class AcceleratorBase(SQLModel):
+    vendor: str | None = Field(
+        description="The manufacturer of the Accelerator.",
+        max_length=NORMAL,
+        schema_extra={"example": "CPU_AMD"},
+    )
+    type: str | None = Field(
+        description="Accelerator type.",
+        max_length=NORMAL,
+        schema_extra={"example": "type"},
+    )
+    model_name: str | None = Field(
+        description="The name of the Accelerator model.",
+        max_length=NORMAL,
+        schema_extra={"example": "Athlon"},
+    )
+    architecture: str | None = Field(
+        description="The accelerator architecture.",
+        max_length=NORMAL,
+        schema_extra={"example": "ARM"},
+    )
+    cores: int | None = Field(
+        description="The number of cores used by the Accelerator.",
+        schema_extra={"example": 8},
+    )
+    memory: int | None = Field(
+        description="The Accelerator memory.",
+        schema_extra={"example": 64},
+    )
```

Comment:
```suggestion
    memory_gb: int | None = Field(
        description="The Accelerator memory (GB).",
        schema_extra={"example": 64},
    )
```
Inferred. In general, make sure people know the unit of the scalar. Preferably also reflect this in the field name, as this makes it easier to know what is happening when working with it as a code object. However, I would also be OK the fieldname is not updated.